build.include

```diff
@@ -41,6 +42,10 @@ parse() {
         ROOT_YARN_CSV="${1#*:}"
         prepare_yarn_root_args $ROOT_YARN_CSV
         shift ;;
+      --arch:*)
+        ARCH="${1#*:}"
+        IMAGE_TAG=$IMAGE_TAG-$TRAVIS_TAG-$ARCH
```

Comment:
default values should not have any 'travis' variable inside

Reply:
+1, since I assume in future TRAVIS_TAG will be set to "" so that GH and Travis built images can be merged into a single tag... and this would default to `$IMAGE_TAG--$ARCH` instead of `$IMAGE_TAG-$ARCH`

Reply:
In order to not conflict with the current build workflow, `--arch` flag is only used in the case of Travis. Once Travis is no more experimental, we can get rid of TRAVIS_TAG flag from all files.

Reply:
build workflow can be updated.

The idea is that we just give a different name that we want to the image

so ./build.sh , by default will produce a next tag

and `TAG=travis-amd64 ./build.sh` should produce a `travis-amd64` image

if issue is around other stuff, we could have variables but it should not be specific to travis.
For example if you want to disable the publication of sha1 image in case of travis, we should have a `DISABLE_SHA1_PUBLISH` flag and not a `TRAVIS_ARG` flag with if (TRAVIS) do this, else do that'

Reply:
Will try excluding TRAVIS_TAG and ARCH variable and set TAG as `next-travis` in case of next build. Will provide more details soon.